### PR TITLE
feature detects for ES6 Symbol;

### PR DESCRIPTION
**Pull request by buhichan**
Adds feature detects for ES6 Symbol. well-known symbols which are not part of es6 such as asyncIterator are not included.

### CODE REVIEW

File: feature-detects/es6/symbol.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "ES6 Symbol",
+  "property": "es6symbol",
+  "notes": [{
```
Comment (rejas) [inline feature-detects/es6/symbol.js]: maybe add a link to the mdn too 
 https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Symbol

File: feature-detects/es6/symbol.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "name": "Official ECMAScript 6 specification",
+    "href": "https://www.ecma-international.org/ecma-262/6.0/#sec-symbol-constructor"
+  }],
+  "polyfills": ["es6shim"],
+  "authors": ["Ron Waldon (@jokeyrhyme)"],
```
Comment (rejas) [inline feature-detects/es6/symbol.js]: you can put your name here :-)

File: feature-detects/es6/symbol.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "ES6 Symbol",
+  "property": "es6symbol",
+  "notes": [{
+    "name": "Official ECMAScript 6 specification",
+    "href": "https://www.ecma-international.org/ecma-262/6.0/#sec-symbol-constructor"
+  }],
+  "polyfills": ["es6shim"],
```
Comment (rejas) [inline feature-detects/es6/symbol.js]: are you sure that polyfill is covering all of symbol too?

File: feature-detects/es6/symbol.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "authors": ["Ron Waldon (@jokeyrhyme)"],
+  "tags": ["es6"]
+}
```
Comment (rejas) [inline feature-detects/es6/symbol.js]: you can also add a caniuse tag here : https://caniuse.com/#feat=mdn-javascript_builtins_symbol_for

File: feature-detects/es6/symbol.js (hunk 2 above)
Comment (buhichan) [reply feature-detects/es6/symbol.js]: I thought here are names of the authors of polyfills, and I'm not one.

File: feature-detects/es6/symbol.js (hunk 4 above)
Comment (buhichan) [reply feature-detects/es6/symbol.js]: I'm not sure where to look for these tags, all other es6 features' tags are just "es6". Maybe ["es6","symbol"] is ok ?

File: feature-detects/es6/symbol.js (hunk 3 above)
Comment (buhichan) [reply feature-detects/es6/symbol.js]: My mistake, es6shim does not cover Symbol.   
Will add this polyfill: https://github.com/medikoo/es6-symbol.
How do you decide which polyfill implementation is the best for this feature?

File: feature-detects/es6/symbol.js (hunk 1 above)
Comment (buhichan) [reply feature-detects/es6/symbol.js]: Added.

File: feature-detects/es6/symbol.js (hunk 2 above)
Comment (buhichan) [reply feature-detects/es6/symbol.js]: Changed.

File: feature-detects/es6/symbol.js (hunk 4 above)
Comment (buhichan) [reply feature-detects/es6/symbol.js]: changed to ["es6","symbol"]

**Comment by buhichan** [comment]
thanks for replying. updated.

**Comment by rejas** [comment]
Thx for the PR and soon a happy new year :-)

**Pull request merged**